dew/data/cmake/dew.cmake

```diff
@@ -7,7 +7,8 @@
 # It is encouraged to check this file into your project's VCS. Doing so will allow your cmake project to easily
 # integrate with Dew.
 #
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.3)
```

Comment:
I'm not sure if 3.2 was actually correct before.. I may be using some cmake features that are introduced in much later versions. I don't really care too much about what the min version we support is, Maybe would be good to support a few versions behind, but it doesn't need to support this old of a cmake.

Reply:
Yeah, it actually looks like 3.12 should be the minimum due to FindPython3 being used instead of the old FindPythonInterp (which has been deprecated).